src/platform/nrfconnect/CHIPDevicePlatformConfig.h

```diff
@@ -270,6 +270,14 @@
 #define CHIP_DEVICE_CONFIG_ENABLE_EXTENDED_DISCOVERY 1
 #endif // CONFIG_CHIP_EXTENDED_DISCOVERY
 
+#ifdef CONFIG_CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING
+#define CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING 1
+#endif // CONFIG_CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING
+
+#ifdef CONFIG_CHIP_DEVICE_CONFIG_DISCOVERY_TIMEOUT_MINUTES
+#define CHIP_DEVICE_CONFIG_DISCOVERY_TIMEOUT_SECS CONFIG_CHIP_DEVICE_CONFIG_DISCOVERY_TIMEOUT_MINUTES * 60
+#endif // CONFIG_CHIP_DEVICE_CONFIG_DISCOVERY_TIMEOUT_MINUTES
```

Comment:
```suggestion
#define CHIP_DEVICE_CONFIG_DISCOVERY_TIMEOUT_SECS (CONFIG_CHIP_DEVICE_CONFIG_DISCOVERY_TIMEOUT_MINUTES * 60)
```
This is always defined for our and Zephyr platform.

Reply:
Done.